definitions/divorce/json/CaseEvent/CaseEvent-deemed-and-dispensed-nonprod.json

```diff
@@ -225,7 +225,7 @@
     "CaseTypeID": "DIVORCE",
     "ID": "IssueAOSOffline_CoRespondent_FromServiceAppRejected",
     "Name": "Issue AOS pack - Co-respondent",
-    "Description": "Answer received",
+    "Description": "Issue AOS pack - Co-respondent",
```

Comment:
Here, should be: "Issue AOS (offline) for Co-Respondent from ServiceApplicationNotApproved"
Unless Jeremy requested to change it that way :)

Reply:
The ACs state,
"Name": "Issue AOS pack - Co-respondent", 
"Description": "Issue AOS pack - Co-respondent",